util/cvanal.c

```diff
@@ -73,9 +73,9 @@ static int32_t cvanal(CSOUND *csound, int32_t argc, char **argv)
         case 's':
           FIND(Str("no sampling rate"))
 #ifdef USE_DOUBLE
-          csound->sscanf(s, "%lf", &sr);
+          sscanf(s, "%lf", &sr);
```

Comment:
I don't know the history here and would have to check, but curious we had a version of sscanf as part of the CSOUND struct API. Was there a reason for this and does moving to sscanf directly cause any potential issues on some platform?

Reply:
There were to do with localisation in CSOUND, LocalizeString, sprintf, sscanf, strtod, strtok_r. Only sscanf and strod were used in plugins. I suppose we need to offer them, but since they did not depend on CSOUND, I thought we could find a better location. I'll put them back for now.